### ISSUE TITLE
Support typed array

### ISSUE THREAD

**Issue by tresdin** [fpirsch/twin-bcrypt#8]
IMHO, since password can be a secure random value, twin-bcrypt should also support typed array as a hashable value.
```
Twin_Bcrypt.hashSync(self.crypto.getRandomValues(new Uint8Array(10)));

**Comment by fpirsch** [comment]
Looks good, since strings are converted to arrays anyway. Arrays could be given in the first place.

**Comment by fpirsch** [comment]
The new feature is implemented by c1ab73b, released as v2.2.0
Thanks for the suggestion.
